packages/account/src/Components/poinc/status/received/received.jsx

```diff
@@ -0,0 +1,16 @@
+import { Icon } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import React from 'react';
+import IconMessageContent from 'Components/icon-message-content';
+
+export const PoincReceived = () => {
+    return (
```

Comment:
I think poincSubmitted is more readable name for this component.. @yauheni-kryzhyk-deriv what do u think?😊

Reply:
I called it 'received', because user will see the message: "We've received your proof of income"
So I believe there is more appropriate and logical. 
Does it make sense?